iceoryx_binding_c/include/iceoryx_binding_c/internal/c2cpp_bridge.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IOX_BINDING_C_INTERNAL_C2CPP_BRIDGE_H_
```

Comment:
please remove trailing underlines from all include guards

Reply:
I really do not like include guards. They are error prone! We need something better - what about `#pragma once` and a nice script which replaces all the `#pragma once` with include guards if someone is insisting using them.

Reply:
For consistency it would be good to have a decision about styling of the include guards. See https://github.com/eclipse/iceoryx/issues/30